src/NuGet.Services.EndToEnd/BasicTests.cs

```diff
@@ -8,16 +8,17 @@
 
 namespace NuGet.Services.EndToEnd
 {
-    public class BasicTests
+    public class BasicTests : IClassFixture<TrustedHttpsCertificatesFixture>
```

Comment:
This class needs a less general name. When we add more tests, we will need clearity on where to put them..